### ISSUE TITLE
Lint/UselessMethodDefinition has a conflict with Rails/LexicallyScopedActionFilter when used with Rails before_action

### ISSUE THREAD

**Issue by rbclark** [rubocop/rubocop#8667]
Given the following simple rails controller:

```ruby
class SessionsController < Devise::SessionsController
  before_action :load_game_for_login, only: %i[new]

  def load_game_for_login
    @game = Game.instance
  end

  def new
    super
  end
end
```

`Lint/UselessMethodDefinition` and `Rails/LexicallyScopedActionFilter` have a disagreement over the proper format of the code.

--------

## Expected behavior

`Lint/UselessMethodDefinition` should not raise a warning when the method is used in a `LexicallySopedActionFilter`.

## Actual behavior

```
app/controllers/registrations_controller.rb:18:3: W: Lint/UselessMethodDefinition: Useless method definition detected.
  def create ...
  ^^^^^^^^^^
```
after running autocorrect:
```
app/controllers/sessions_controller.rb:5:3: C: Rails/LexicallyScopedActionFilter: new is not explicitly defined on the class.
  before_action :load_game_for_login, only: %i[new]
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

## Steps to reproduce the problem

1. Create a basic controller with a before_action and a definition for `new` in its parent class. (as seen in the example)
2. Run rubocop with both the listed cops enabled, and run autocorrect which will cause the above two errors to show up.

## RuboCop version

```
$ bundle exec rubocop -V
0.90.0 (using Parser 2.7.1.4, rubocop-ast 0.3.0, running on ruby 2.7.1 x86_64-darwin19)
```

**Comment by JunichiIto** [comment]
I had the same problem too. This issue might be duplicated with https://github.com/rubocop-hq/rubocop/issues/8667

**Comment by marcandre** [comment]
FWIW, I am not too sure what `Rails/LexicallyScopedActionFilter` is trying to prevent 😅 IIUC, if an action is inherited and there's no action filter applying to it, that's fine, but if there is an action filter then it's not? Having to write useless code to satisfy a cop seems wrong, better turn the cop on or `# rubocop:disable`  it as a way of saying "hey, it's ok, it's inherited". If you have multiple filters on the same inherited action,  you have to do that on each filter though...

**Comment by koic** [comment]
It will likely be resolved by rubocop/rubocop-rails#1500.
